docker run \
  -v android:/home/build \
  hashbang/os bash -c "cd base; repo diff -u"

project build/make/
diff --git a/build/make/target/product/core.mk b/build/make/target/product/core.mk
index c72bb2f26..5fb0ec350 100644
--- a/build/make/target/product/core.mk
+++ b/build/make/target/product/core.mk
@@ -60,6 +60,7 @@ PRODUCT_PACKAGES += \
     Telecom \
     TeleService \
     Traceur \
+    Updater \
     VpnDialogs \
     vr \
     MmsService

project packages/apps/Updater/
diff --git a/packages/apps/Updater/res/values/config.xml b/packages/apps/Updater/res/values/config.xml
index b9fc22f..b36dfeb 100644
--- a/packages/apps/Updater/res/values/config.xml
+++ b/packages/apps/Updater/res/values/config.xml
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="url" translatable="false">https://seamlessupdate.app/</string>
+    <string name="url" translatable="false">https://os.hashbang.sh/</string>
 </resources>
